### PR TITLE
[15.0] [MIG] website_sale_product_brand: Migration to 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT36604

Please wait to see if a PR is accepted to make the options inheritable. As soon as it is done I paste it in this thread.

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
This is the PR to improve inheritance https://github.com/odoo/odoo/pull/92155

**Comment by chienandalu** [comment]
Please add #660

**Comment by chienandalu** [comment]
> This is the PR to improve inheritance [odoo/odoo#92155](https://github.com/odoo/odoo/pull/92155)

Already merged :smile:

**Comment by CarlosRoca13** [comment]
Done! @chienandalu

**Review by chienandalu — APPROVED** [review]
Tested :+1: Thanks!

File: website_sale_product_brand/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.model
+    def _search_get_detail(self, website, order, options):
+        res = super()._search_get_detail(website, order, options)
+        domain = res["base_domain"]
+        # Need to get the context from request because if we get it from 'env' we will
+        # loose the brand_id key.
+        # TODO: Change this context by extending the options when odoo makes it
```
Comment (pedrobaeza) [inline website_sale_product_brand/models/product_template.py]: This should change now that the PR is merged, isn't it?

Comment (CarlosRoca13) [reply website_sale_product_brand/models/product_template.py]: It is changed https://github.com/OCA/e-commerce/pull/648/commits/a66034ee9107a03ff9eacdab938dd5effd32bd61#diff-2bebf7bbe0ca4477095ab7d48844aa37f7e29a7e3dccf44af10be4bd5057a231L14-L22

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**